src/xrpld/app/tx/detail/VaultClawback.cpp

```diff
@@ -154,7 +154,7 @@ VaultClawback::doApply()
         return tefINTERNAL;  // LCOV_EXCL_LINE
 
     auto const mptIssuanceID = (*vault)[sfShareMPTID];
-    auto const sleIssuance = view().read(keylet::mptIssuance(mptIssuanceID));
+    auto const sleIssuance = view().read(keylet::mptIssuance(*mptIssuanceID));
```

Comment:
Rather than using the `*` operator every time you access `mptIssuanceID`, you could do it when you do the initial lookup:
```suggestion
    auto const mptIssuanceID = *(*vault)[sfShareMPTID];
    auto const sleIssuance = view().read(keylet::mptIssuance(mptIssuanceID));

```

Reply:
thanks, implemented in https://github.com/XRPLF/rippled/pull/5652/commits/6893015759ad8b7041611551c453b6efd9ac9051 (with extra parents for readability)